### PR TITLE
[COM-134] checkbox component

### PR DESCRIPTION
**Pull request by dawinValenzuela**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://impulsumvc.atlassian.net/browse/COM-134

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

**Storybook Link** (Link to a local storybook url to review new functionality)
http://localhost:8080/?path=/story/components-primitives-checkbox--all

### CODE REVIEW

File: packages/components/src/components/primitives/Checkbox/types.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { CheckboxProps as ChakraCheckboxProps } from '@chakra-ui/react';
+
+export const CheckboxSizes = ['sm', 'md', 'lg'] as const;
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Checkbox/types.ts:3]: sizes are defined as `s`, `m` and `l` in figma

File: packages/components/src/theme/ComposerTheme/Components/Checkbox.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      label: {
+        ...theme.textStyles['text-body-meta-regular'],
+      },
+    }),
+    md: ({ theme }) => ({
+      label: {
+        ...theme.textStyles['text-body-regular'],
+      },
+    }),
+    lg: ({ theme }) => ({
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Checkbox.ts:50]: ditto

File: packages/components/src/components/primitives/Checkbox/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './Checkbox';
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Checkbox/index.ts:1]: missing re-export in the parent folder

**Review by tkanzakic — COMMENTED** [review]
just a couple of minor comments, good work!

File: packages/components/src/components/primitives/Checkbox/Checkbox.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      },
+    },
+  ];
+
+  return (
+    <VStack divider={<StackDivider borderColor="gray.200" />} spacing={4}>
+      {values.map((val, key) => (
+        <HStack key={key}>
+          {CheckboxSizes.map((size) => (
+            <Checkbox key={size} size={size} {...val?.props}>
```
[the remark this answers is not in the document]
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Checkbox/Checkbox.stories.tsx]: done!

File: packages/components/src/components/primitives/Checkbox/types.ts (hunk 1 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Checkbox/types.ts]: done!

File: packages/components/src/theme/ComposerTheme/Components/Checkbox.ts (hunk 2 above)
Comment (dawinValenzuela) [reply packages/components/src/theme/ComposerTheme/Components/Checkbox.ts]: done!

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**